nodeinfo.go

```diff
@@ -21,7 +21,12 @@ import (
 	"go.opencensus.io"
 )
 
-func createNodeInfo(nodeName string) *commonpb.Node {
+// NodeWithStartTime creates a node using nodeName and derives:
+//  Hostname from the environment
+//  Pid from the current process
+//  StartTimestamp from the start time of this process
+//  Language and library information.
+func NodeWithStartTime(nodeName string) *commonpb.Node {
```

Comment:
I would say I prefer the old method name :) Not only start_time is derived from env vars.

Reply:
Gotcha. The reason why I changed it is because all the other variables can be retrieved alright from the environment but we don't expose startTime so it is the most important one.